keras/layers/normalization.py

```diff
@@ -111,15 +111,35 @@ def build(self, input_shape):
                                          regularizer=self.gamma_regularizer,
                                          constraint=self.gamma_constraint)
         else:
-            self.gamma = None
+            # mxnet backend does not take gamma symbol as None, initialize as ones
+            # and set it as not trainable
+            if K.backend() == 'mxnet':
+                self.gamma = self.add_weight(shape=shape,
+                                             name='gamma',
+                                             initializer=self.gamma_initializer,
```

Comment:
is gamma_initializer creating all ones and beta_initializer all zeros?
As we were discussing offline, how are you making sure this parameter is not learnt?
What is the effect on model saving and loading - if users are setting scale/center to None, what is saved in trained model?